ui/components/app/confirm-page-container/confirm-page-container-header/confirm-page-container-header.component.js

```diff
@@ -64,7 +72,19 @@ export default function ConfirmPageContainerHeader({
             </span>
           </div>
         )}
-        {isFullScreen ? null : <NetworkDisplay />}
+        {isFullScreen ? null : (
+          <PickerNetwork
+            label={
+              networkType === NETWORK_TYPES.RPC
+                ? nickname ?? t('privateNetwork')
+                : t(getNetworkLabelKey(networkType))
+            }
+            backgroundColor={BackgroundColor.transparent}
+            avatarNetworkProps={{
+              backgroundColor: testNetworkBackgroundColor,
+            }}
```

Comment:
We can also hide the caret icon using style utility props and the `iconProps` prop object
```suggestion
            }}
            iconProps={{
              display: Display.None,
            }}
```